src/components/DuosHeader.js

```diff
@@ -14,7 +116,8 @@ class DuosHeader extends Component {
     this.state = {
       showSupportRequestModal: false,
       hover: false,
-      dacChairPath: '/chair_console'
+      dacChairPath: '/chair_console',
```

Comment:
Not for this PR, but https://broadworkbench.atlassian.net/browse/DUOS-1045 should address the bug that we aren't dynamically checking for the correct console path here.